Create an interface for accepting a new to-do (#108)
pupilfirst/wd201-scripts: todo-user-interface/interface-for-new-to-do/README.md

# Text

> In the previous lesson, you've learned about the key areas which you should keep in mind while designing a webpage. Now, it's time to use those principals to convert the To-Do manager design into an webpage. Let's get started.

Reviewer: ```suggestion
In the previous lesson, you've learned about the key areas which you should keep in mind while designing a webpage. Now, it's time to use those principles to convert the To-Do manager design into a webpage. Let's get started.
```

# Script

> In this lesson, we will start with our To-Do manager application, to convert the design into an webpage.

Reviewer: ```suggestion
In this lesson, we will start with our To-Do manager application, to convert the design into a webpage.
```

```

Then, we will add the **TailwindCSS** into the `<head>` section of this HTML file. Previously, we've already introduced you to TailwindCSS, a utility-first CSS framework, packed with tons of CSS classes, which can be composed to build any webpage.
```html
<head>
  <title>TO-DO Manager</title>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <script src="https://cdn.tailwindcss.com"></script>
</head>
```

To check, if the Tailwind CSS is loaded properly, add a simple heading (h1) there:
```html
<body>
  <h1 class="text-2xl font-semibold text-gray-600">
    My Todo-list
  </h1>
</body>
```
Great! It's working.

### Now, let's define the Layout
To define the layout, we will use Grids from TailwindCSS. You can read their [documentation](https://tailwindcss.com/docs/grid-template-columns) for more information.
As per the design image, the Todo-Manager is positioned at the center of the page, with even blank spaces on both left and right sides.
```html
<body>
  <div class="grid grid-cols-6">
    <div class="col-start-3 col-span-2">
      <h1 class="text-2xl font-semibold text-gray-600 mt-20">
        My Todo-list
      </h1>
    </div>
  </div>
<body>
```

> Here, `grid-cols-6` divides the whole screen into six columns, and `col-start-3` defines that our primary container will start from **third** column. `col-span-2` says the browser that the container will be 2 column wide. Let's see the output in browser.

Reviewer: ```suggestion
Here, `grid-cols-6` divides the whole screen into six columns, and `col-start-3` defines that our primary container will start from **third** column. `col-span-2` says the browser that the container will be 2 columns wide. Let's see the output in the browser.
```
